lib/device_api/android.rb

```diff
@@ -17,7 +17,7 @@ module Android
     def self.devices
       ADB.devices.map do |d|
         if d.keys.first && !d.keys.first.include?('?')
-          DeviceAPI::Android::Device.create( self.get_device_type(d.keys.first), { serial: d.keys.first, state: d.values.first } )
+          DeviceAPI::Android::Device.create( self.get_device_type(d), { serial: d.keys.first, state: d.values.first } )
```

Comment:
Line is too long. [119/80]<br>Redundant curly braces around a hash parameter.<br>Redundant `self` detected.<br>Space inside parentheses detected.